docs/api.md

```diff
@@ -1127,17 +1127,17 @@ log.info('this will be written to /tmp/debug.stream.out and /tmp/info.stream.out
 log.fatal('this will be written to /tmp/debug.stream.out, /tmp/info.stream.out and /tmp/fatal.stream.out')
 ```
 
-In order for `multistream` to work, the log `level` __must__ be set to the lowest `level` used in the streams array.
+In order for `multistream` to work, the log `level` __must__ be set to the lowest `minLevel` used in the streams array.
 
 #### Streams
 
 Stream or array of streams. Each stream may be a `stream` (`DestinationStream`) or `object` (`StreamEntry`) with the following properties:
 
 + `stream`: A Node.js stream (Default to `process.stdout`)
-+ `level`: The log level of the stream (Default to `'info'`)
++ `minLevel`: The log level of the stream (Default to `'info'`)
 + `maxLevel`: The max log level of the stream (Optional)
 
-`level` and `maxLevel` are used to filter the logs using the following rule: `stream.level <= log.level < stream.maxLevel`. If `maxLevel` is not specified it will not be used.
+`minLevel` and `maxLevel` are used to filter the logs using the following rule: `stream.minLevel <= log.level < stream.maxLevel`. If `maxLevel` is not specified it will not be used.
```

Comment:
This is what confuses me about `maxLevel`. Setting `minLevel` is inclusive, but `maxLevel` is exclusive. This is inconsistent and confusing. But if `maxLevel` is _inclusive_ then the feature does not accomplish the goal of allowing only a _single_ level to reach the stream. This is why I was thinking an allow list would be better (while recognizing that it has a few problems).

In order to get consistent behavior, I think the stream should accept `onlyLevel: 'info'` as an option.

I am desiring this feature myself, but I fear we are getting deep into "hack an API" land. What are your thoughts @mcollina @davidmarkclements ?

Reply:
@jsumners How could I archieve the same result with `onlyLevel`? Check my original problem:

> > I actually have a pretty stream transport and a file stream transport (that is only working in production). I would like the pretty stream to use the stdout for all levels except errors, I cannot use dedupe as this would prevent logs to be logged on fileStream in production. If I remove dedupe I would have the logs duplicated on error and stdout, If i add dedupe, in production, I would miss the logs in file. I just would like to have an option in pretty stream to say log error levels to stderr and others to stdout, or create a custom stream to handle this

If I set onlyLevel to error stream I would have the same problem I have now that the logs are duplicated, and if I use the dup flag I would miss the logs in the file stream

I think this is the best solution to have more control of the stream levels

Reply:
> If I set onlyLevel to error stream I would have the same problem I have now that the logs are duplicated, and if I use the dup flag I would miss the logs in the file stream

I don't understand. This should allow for sending desired levels to specific and all things to others:

```js
const log = pino({ level: 'info' }, pino.multistream([
  { onlyLevel: 'error', stream: errorStream },
  { onlyLevel: 'debug', stream: debugStream },
  { level: 'info', stream: pino.destination() } // normal output stream
]))
```

Reply:
Ok so in my case i would like to print `silly`, `debug`, `info` and `warn` to stdout and `error`, `fatal` to stderr, I should add 4 stream entries for the stdout and 2 for the stderr plus one for the file stream.